spyder_line_profiler/__init__.py

```diff
@@ -1,14 +1,14 @@
 # -*- coding: utf-8 -*-
+# ----------------------------------------------------------------------------
+# Copyright © 2022, Spyder Line Profiler 5
```

Comment:
Maybe, for the files preserved in the API switch, this could be something like:
``` python
# Copyright © 2013 - Spyder Line Profiler contributors
```

 and,  for the new files, something like:

``` python
# Copyright © 2022 - Spyder Line Profiler contributors
```

?

Reply:
I left the copyright everywhere the same and replaced the new ones with 2022. I think almost no file stayed the same.